src/ray/core_worker/core_worker.cc

```diff
@@ -235,9 +234,12 @@ CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
   if (direct_task_receiver_ != nullptr) {
     direct_task_receiver_->Init(client_factory, rpc_address_);
   }
+  plasma_notifier_.reset(new ObjectStoreNotificationManager(io_service_, store_socket));
 }
 
 CoreWorker::~CoreWorker() {
+  // Possibly account for shutdown
+  plasma_notifier_.release();
```

Comment:
Is this just shared pointer release? won't this be called in just normal shared pointer destructor?

Reply:
Yeah, I think it is fine to remove this.